files-community/Files
# Fix mixed fonts when custom font is being used

**Resolved / Related Issues**
   Closes #12026...

**Validation**
How did you test these changes?
   1. Open app
   2. Open Settings -> Advanced -> Edit Settings File
   3. Change value for key "AppThemeFontFamily" to another installed font (e.g., SpaceMono NF)
   4. Save user_settings.json
   5. Restart app
   6. Open various menu flyouts and sub-menus
   7. Observe fonts are the same between them

**Screenshots (optional)**

![Screenshot 2023-09-22 223151](https://github.com/files-community/Files/assets/70173112/1de7e4b9-6922-479e-a009-df199c249c06)
![Screenshot 2023-09-22 223147](https://github.com/files-community/Files/assets/70173112/a86edb3d-feb1-4460-b3dd-4b75955da699)
![Screenshot 2023-09-22 223138](https://github.com/files-community/Files/assets/70173112/0649fcba-eeac-4bcd-9c0a-1e2379715814)
![Screenshot 2023-09-22 223130](https://github.com/files-community/Files/assets/70173112/5c3c625e-6151-4072-822f-e3221277ce95)
![Screenshot 2023-09-22 223241](https://github.com/files-community/Files/assets/70173112/1e222dbd-72af-4426-a0a0-f53da2374290)
![Screenshot 2023-09-22 223237](https://github.com/files-community/Files/assets/70173112/321a91c5-026c-459f-87f3-58aa88bf375d)
![Screenshot 2023-09-22 223232](https://github.com/files-community/Files/assets/70173112/97961dfe-d99a-4188-82e8-060d3eb77aa2)
![Screenshot 2023-09-22 223218](https://github.com/files-community/Files/assets/70173112/b27efee6-42ab-478a-8554-e284136e53d6)
![Screenshot 2023-09-22 223200](https://github.com/files-community/Files/assets/70173112/1b435102-6dfb-405b-bfcf-0455a506307b)
![Screenshot 2023-09-22 223155](https://github.com/files-community/Files/assets/70173112/489d9cf6-cbdb-4b33-a257-b265717e7a8c)
